master/buildbot/reporters/http.py

```diff
@@ -0,0 +1,79 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from buildbot import config
+from buildbot.reporters import utils
+from buildbot.util import service
+
+from twisted.internet import defer
+from twisted.python import log
+
+# use the 'requests' lib: http://python-requests.org
+try:
+    import txrequests
+    assert txrequests
```

Comment:
What exactly this line is supposed to do?  If the import is successful, then it's gonna be OK, if not, then `ImportError` would be thrown, AFAIU.